### PR TITLE
MGMT-4580 InstallEnv should only accept a single optional SSH key

### PR DESCRIPTION
**Pull request by nmagnezi**
That key will get injected into the discovery image for debug proposes

### CODE REVIEW

**Comment by nmagnezi** [comment]
/hold
testing

File: docs/crds/installEnv.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ spec:
   proxy:
     httpProxy: http://11.11.11.33
     httpsProxy: http://22.22.22.55
+  sshAuthorizedKey: 'ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDphFEgnLEQknipClTiSoHpvu078bd0aTSp5WBfXvSC5nAqkprUcF/rB+iNne/MpjSONEqiUyHZayvRvVgEGfRUucAUQ89o36aJmMT8N1U85uNuPNSHVuNogyAi/pmh29C6vfs1B1qGSZRA4P/FvYn7+NdQgZQzqHxkWUDj3BqMby3+b2VKOKQ4SDHXaBSrR9BigWEWCwFJeHgTFAaEZ8p8XpPhEOhbfn9ZZ47Myq1LqfP4gw+62dBhYBNI7r44PnWbLczNLTHYUZXyappt9NBBrcwHtG+3NogDCt0eJkSYydeJh5wPQYN50kvZNJdXrMtGx9WDutTr5famFRmqcxmD'
```
Comment (YuviGold) [inline docs/crds/installEnv.yaml:17]: We shouldn't have secrets on GitHub

Comment (nmagnezi) [reply docs/crds/installEnv.yaml:17]: It is just an example key in an example YAML (same as [clusterDeployment](https://github.com/openshift/assisted-service/blob/master/docs/crds/clusterDeployment-SNO.yaml#L25))
I can change it to `"your_pub_key_here"`

Comment (rollandf) [reply docs/crds/installEnv.yaml:17]: This is public key, I don't think it is a problem.
Here is yours:
https://github.com/YuviGold.keys

Comment (YuviGold) [reply docs/crds/installEnv.yaml:17]: > It is just an example key in an example YAML (same as [clusterDeployment](https://github.com/openshift/assisted-service/blob/master/docs/crds/clusterDeployment-SNO.yaml#L25))
> I can change it to `"your_pub_key_here"`

Okay, I missed that it is documentation.
I still think that a placeholder would suites better

Comment (nmagnezi) [reply docs/crds/installEnv.yaml]: Done (also for clusterDeployment)

**Review by YuviGold — APPROVED** [review]
LGTM

**Review by filanov — COMMENTED** [review]
please check that ssh is optional in backend as well

**Pull request merged**